src/main/java/appeng/core/lib/features/BlockDefinition.java

```diff
@@ -85,4 +89,13 @@ public boolean isSameAs( Object other )
 		}
 	}
 
+	public IItemDefinition buildItemBlock( FeatureFactory factory){
```

Comment:
Storing them in the FeatureFactory would indeed be better. But thay already contain build-related stuff (setItem()) and are not yet complete when they are build() because the ItemBlock is missing.